sal_interfaces/FiberSpectrograph/FiberSpectrograph_Commands.xml

```diff
@@ -1,39 +1,51 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
-<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
-<SALCommand>
-    <Subsystem>FiberSpectrograph</Subsystem>
-    <Version>3.7.1</Version>
-    <Author></Author>
-    <EFDB_Topic>FiberSpectrograph_command_captureSpectImage</EFDB_Topic>
-    <Alias>captureSpectImage</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Description>Capture a spectrum for integration time specified</Description>
-    <item>
-        <EFDB_Name>integrationTime</EFDB_Name>
-        <Description>The detector in the spectrometer can capture light for a length of time specified by integration time</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>second</Units>
-        <Count>1</Count>
-    </item>
-    <item>
-        <EFDB_Name>imageType</EFDB_Name>
-        <Description>Type of image. For example, Standard exposure.</Description>
-        <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
-        <Units>unitless</Units>
-        <Count>1</Count>
-    </item>
-    <item>
-        <EFDB_Name>lamp</EFDB_Name>
-        <Description>Light used for measurement.</Description>
-        <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
-        <Units>unitless</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand></SALCommandSet>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+    <SALCommand>
+        <Subsystem>FiberSpectrograph</Subsystem>
+        <Version>3.7.1</Version>
+        <Author/>
+        <EFDB_Topic>FiberSpectrograph_command_expose</EFDB_Topic>
+        <Alias>expose</Alias>
+        <Device/>
+        <Property/>
+        <Action/>
+        <Value/>
+        <Description>Capture an image from the spectrograph.</Description>
+        <item>
+            <EFDB_Name>duration</EFDB_Name>
+            <Description>The duration of the exposure to take.</Description>
+            <IDL_Type>float</IDL_Type>
+            <Units>second</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>type</EFDB_Name>
+            <Description>Type of image. What does this mean?????</Description>
+            <IDL_Type>string</IDL_Type>
+            <IDL_Size>256</IDL_Size>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>lamp</EFDB_Name>
+            <Description>Light used for measurement. Is this relevant???</Description>
```

Comment:
You're correct that this is essentially an "IMG_TYPE" keyword. A config file isn't the correct place for it as it can change from exposure-to-exposure.
We have to do a little more testing of the hardware to figure out where these will be mounted. I think it's best to leave it as a user-input for now (like IMG_TYPE with the take images command). In fact, we ought to think of just calling it IMG_TYPE.
John and myself need to have a talk about meta-data, this is the type of thing that will be discussed.

Reply:
@patrickingraham Isn't the field named `type` the field that you are talking about (the one that maps to IMGTYPE)

I was suggesting we keep the field named `type` and get rid of the field named `lamp` -- making *that* data based on which spectrograph is in use.

Reply:
We clarified this in an offline conversation.